Mathlib/AlgebraicTopology/Quasicategory.lean

```diff
@@ -55,4 +55,25 @@ lemma Quasicategory.hornFilling {S : SSet} [Quasicategory S] ⦃n : ℕ⦄ ⦃i
 instance (S : SSet) [KanComplex S] : Quasicategory S where
   hornFilling' _ _ σ₀ _ _ := KanComplex.hornFilling σ₀
 
+open SimplicialObject SimplexCategory in
+lemma quasicategory_of_filler (S : SSet)
```

Comment:
Is it possible to extract from this proof the extensionality property that a morphism from a horn is determined by its restriction to suitable faces?

Reply:
Good suggestion. Done.